### DISCUSSION TITLE
Welcome to Sofa Time Discussions!

### DISCUSSION THREAD

**Discussion by javikr** [javikr/sofatime#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bsmn77k4zb-sketch** [comment]
Hello!  I am here to learn more about sofatime & get tips on how to make the most of this wonderful app.

**Reply by bo7ussein** [reply]
أريد تحويل حسابي الى حساب ذكر وليس انثى 

I want to change my account from a female account to a male account

**Comment by alkhelaiwimoh-png** [comment]
The homepage of my profile doesn't show my recent movie and TV show viewings; instead, it displays them in order from oldest to newest. I have to scroll right to see my recent viewings. I don't know if this problem is only with the Arabic language or with the app itself. I hope this issue will be resolved so that my recent viewings appear automatically on my profile without needing to scroll right.

**Comment by h7yzvjbz95-cmyk** [comment]
Im interested about the achievement 
I need 100 to follow me
And put like in my list so if anyone can do it 
<img width="660" height="1434" alt="IMG_7210" src="https://github.com/user-attachments/assets/b319e3e7-1ab8-4a43-9841-deb71d39ceb9" />
